content/ss20/bachelor/b4-energy-matters/video.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "Video"
+
+# These properties may be removed if you don't need them
+
++++
+
+{{< back >}}
+
+
+
+{{< video src="energy_matters_showtime.mp4" width="700">}}
```

Comment:
Könnt ihr das Video bitte noch in die [HTW Mediathek](https://mediathek.htw-berlin.de) übertragen und mit [dem entsprechenden Shortcode](https://github.com/htw-imi-showtime/showtime-website#HTW-Mediathek-Player) einbinden? :)

Reply:
Bitte kurz bescheid wenn das erledigt ist. Weiter sieht es so aus, als ob Sie den größten Teil des Textes gelöscht haben - stimmt das so?

Reply:
Haben die video file entfernt und vie HTW Mediathek eingebettet. Der Text ist mit Absicht kürzer geworden und mit mehr Grafiken versehen worden, da die Seite vorher ja sehr Textlastig war. Im Präsentationsvideo ist außerdem alles relevante enthalten. Danke :)